nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -362,17 +362,26 @@ void reportInitErrorOnField(Symbol symbol, VisitorState state, Description.Build
       return;
     }
     Tree tree = getTreesInstance(state).getTree(symbol);
+
+    String fieldName = symbol.toString();
+
+    if (symbol.enclClass().getNestingKind().isNested()) {
+      String flatName = symbol.enclClass().flatName().toString();
+      int index = flatName.lastIndexOf(".") + 1;
+      fieldName = flatName.substring(index) + "." + symbol;
```

Comment:
nit: Why `+ symbol` and not `+ fieldName`? (From line 366), seems like this will force an extra call to `.toString()` that is not needed.